site/src/pages/DeploySettingsPage/Option.tsx

```diff
@@ -91,11 +91,11 @@ export const OptionValue: FC<OptionValueProps> = (props) => {
                 }}
               >
                 {isEnabled && (
-                  <CheckCircleOutlined
+                  <LabelImportant
                     css={(theme) => ({
                       width: 16,
                       height: 16,
-                      color: theme.palette.success.light,
+                      color: theme.palette.info.light,
```

Comment:
(non-blocking): I wonder if we can use colour to denote 'invalid' experiments here. Maybe `theme.palette.warning`?

Reply:
Absolutely, but we need to pass in invalid experiments to this component.
It's trivial to add, but the abstraction of this component is already bleeding so we'd need to refactor it a bit to make it clean. Perhaps not worth the effort unless someone from FE can propose a simpler solution.

Reply:
Yeah, this component is already handling a lot of branches, so I'm a little wary of passing the list of invalid experiments in directly

One option that comes to mind would be to update the component's props to include an optional `isValid` boolean prop. It would default to true, but depending on whether it's true/false, that would affect the icon output. But I imagine that would require some updates to the `OptionsTable` component

One thing I do want to call out is that if we do want to draw attention to invalid experiments, we will need to make sure that we're doing more than changing the color, to stay accessible to colorblind folks. Changing both the icon type and the color should be good, though

Reply:
I think I'm just going to stick with the warning alert for now; the extra complexity this brings with it is probably not worth it.

Reply:
![image](https://github.com/coder/coder/assets/373762/6d85bd01-69e2-4436-af3f-411979fe631d)

I've changed the color to white; I think this is clear enough - from a visual perspective, we're just indicating these values are set, and not indicating from a color-perspective if their values are valid or not.

What do you folks think?

Reply:
I'm personally fine with that for now. We can add fanciness later if needed.

Reply:
Yeah, I think it looks good